### PR TITLE
change grpc bidi-stream error handling and change grpc API interface

### PR DESCRIPTION
**Pull request by kpango**
Signed-off-by: kpango <kpango@vdaas.org>

### Description:
change grpc bidi-stream error handling and change grpc API interface for more informative payload for each stream response

### Environment:

- Go Version: 1.15.2
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apis/proto/v1/gateway/vald/vald.proto [hunk 1]
```diff
@@ -87,7 +87,8 @@ service Vald {
     option (google.api.http).delete = "/remove/{id}";
 
   }
-  rpc StreamRemove(stream payload.v1.Object.ID) returns (stream payload.v1.Object.Location) {}
+  rpc StreamRemove(stream payload.v1.Object.ID) 
+      returns (stream payload.v1.Object.Location) {}
```
Comment (rinx) [inline apis/proto/v1/gateway/vald/vald.proto:91]: Good. :+1:

How about the other stream methods in vald.proto?

Comment (rinx) [inline apis/proto/v1/gateway/vald/vald.proto:91]: StreamResponses is not used in vald.proto?

Comment (kpango) [reply apis/proto/v1/gateway/vald/vald.proto:91]: I'm thinking about it, should i fix v1/gateway/vald/vald.proto???
may be gateway/vald will be duplicated I think we shouldn't fix anything about it... may be...:)

Comment (rinx) [reply apis/proto/v1/gateway/vald/vald.proto:91]: maybe you mean "deprecated". 
i see. okay.

**Pull request merged**